SeriesMaster-Frontend/App/components/CardImage.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components/native";
+import PropTypes from "prop-types";
+
+const StyledCardImage = styled.Image`
+  height: 125px;
+  width: 125px;
```

Comment:
These should be set by a grid or flexbox, could be in a separate PR no probs

Reply:
Yes, I will try to add it when I generate my Elements from the JSON server. Thanks!